cmd/chartsvc/handler.go

```diff
@@ -61,34 +63,127 @@ type rel struct {
 	Links selfLink    `json:"links"`
 }
 
-// listCharts returns a list of charts
-func listCharts(w http.ResponseWriter, req *http.Request) {
+type meta struct {
+	TotalPages int `json:"totalPages"`
+}
+
+type count struct {
+	Count int `json:"count"`
```

Comment:
I believe this doesn't need the `json` tag since we never marshal it, right? Not hugely important, but I might accidentally think that we do marshal it based on this.